contrib/spec/podman.spec.in

```diff
@@ -52,7 +52,9 @@ ExclusiveArch: aarch64 %{arm} ppc64le s390x x86_64
 # The COPR process will uncomment this
 #BuildRequires: golang-bin
 #
+%if 0%{?fedora} > 28
```

Comment:
Just do a negative RHEL test here.  Then any distro that uses build_rpm.sh would not be effected.
SUSE and perhaps others might use this script.  Don't worry about old versions of Fedora.

Reply:
Resolved here: https://github.com/containers/libpod/pull/4587/files#diff-3542d73edda6203e1915e86cb80b743fR55 Thanks for the comment.